internal/integration_test/integration_test.go

```diff
@@ -35,7 +35,10 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-const maxUDPPacketSize = 64 * 1024
+const (
+	maxUDPPacketSize = 64 * 1024
+	timeout          = 5 * time.Minute
```

Comment:
"timeout" is too generic.

```suggestion
	natTimeout          = 5 * time.Minute
```